xwiki-rendering-syntaxes/xwiki-rendering-syntax-event/src/main/java/org/xwiki/rendering/internal/renderer/event/EventsChainingRenderer.java

```diff
@@ -431,9 +431,9 @@ private void printMacroData(String eventName, String name, Map<String, String> p
         StringBuilder parametersBuffer = new StringBuilder();
         for (Iterator<Map.Entry<String, String>> paramsIt = parameters.entrySet().iterator(); paramsIt.hasNext();) {
             Map.Entry<String, String> entry = paramsIt.next();
-            parametersBuffer.append(entry.getKey()).append("=").append(entry.getValue());
+            parametersBuffer.append(entry.getKey()).append(String.valueOf('=')).append(entry.getValue());
```

Comment:
This is not what this Sonar issue is about. The point is to use `append(char)` instead of `append(String)`, no need to go trough `String.valueOf`.